### PR TITLE
Added support for a second inline binding format

### PR DESCRIPTION
**Pull request by jshook**
This provides an alternative form for when the templating language conflicts with other tools, like mustache or zola.

### CODE REVIEW

**Review by jeffbanks — APPROVED** [review]
lgtm

File: adapter-http/src/main/resources/http.md [hunk 1]
```diff
@@ -203,7 +203,7 @@ an op template like `space: {space}`, OR use an inline op field in your op
 template like `space: {(ThreadNumToInteger())}`.
 
 You can use any binding function you want for the space op field. However,
-if you were to assign it something like "space: {{Identity()}}" you would
+if you were to assign it something like "space: {(Identity()}" you would
```
Comment (msmygit) [inline adapter-http/src/main/resources/http.md:206]: `#trivial` I see there is an open paranthesis "`(`" but, don't see the equivalent closing paranthesis "`)`" and just wanted to call your attention here to validate if this is indeed the right format.

Comment (jshook) [reply adapter-http/src/main/resources/http.md:206]: Thanks, I'll fix this.

Comment (jshook) [reply adapter-http/src/main/resources/http.md:206]: fixed in current code to be pushed.

**Pull request merged**